<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QLineEdit" name="customColorEdit1"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="customColorEdit3"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="xAxisDefaultLabel">
     <property name="text">
      <string>Default X-Axis Data</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QComboBox" name="xAxisDefaultBox"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="customColorEdit4"/>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="customColorEdit5"/>
   </item>
   <item row="14" column="1">
    <widget class="QPushButton" name="applyButton">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="colorComboBox"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="colorPaletteLabel">
     <property name="text">
      <string>Color Palette </string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="dataTransformLabel">
     <property name="text">
      <string>Data Transform</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="yAxisTitleLabel">
     <property name="text">
      <string>Y-Axis Title</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <spacer name="dataTransformSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="graphTitleEdit"/>
   </item>
   <item row="10" column="0">
    <widget class="QComboBox" name="dataTransformBox"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="customColorCodeLabel">
     <property name="text">
      <string>Custom Color Codes</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="graphTitleLabel">
     <property name="text">
      <string>Graph Title</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLineEdit" name="yAxisTitleEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="xAxisTitleLabel">
     <property name="text">
      <string>X-Axis Title</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="dataSortingLabel">
     <property name="text">
      <string>Data Sorting</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QComboBox" name="dataSortingBox"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="customColorEdit2"/>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="xAxisTitleEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
